Source/WebCore/PAL/pal/spi/cocoa/QuartzCoreSPI.h

```diff
@@ -361,3 +361,7 @@ extern NSString * const kCASnapshotOriginY;
 extern NSString * const kCASnapshotTransform;
 extern NSString * const kCASnapshotTimeOffset;
 #endif
+
+@interface CARemoteExternalEffect (Radar_160881286)
```

Comment:
You'll probably want to include `CARemoteEffectPrivate.h` for `USE(APPLE_INTERNAL_SDK)`, and declare `@interface CARemoteExternalEffect : CARemoteEffect;` for `!USE(APPLE_INTERNAL_SDK)` first.

Reply:
Also, maybe leave a FIXME to clean up further when the interface is available in more SDKs?